docs/articles/posts/20250314_gen_ai_search_engagement.md

```diff
@@ -0,0 +1,74 @@
+### *Don’t forget to document it*
+If you are already doing it, why not document it! The quality framework includes comms and marketing. We suggested to Will that he fill in the [Comms and Marketing section](https://nhsd-confluence.digital.nhs.uk/display/DAT/DS_299%3A+Comms+and+Marketing) to create a marketing plan and gave advice on how to do so, and what activities could be reasonable for it; for example Will had been “keeping an eye out for suitable chances to present work” and that is a valid marketing activity. 
+We also suggested Will could write an article on his work ([which he went and did!](https://nhsengland.github.io/datascience/articles/2024/12/31/LLM-as-a-Judge/)), share it on the website and cross-share on other platforms such as LinkedIn to reach a wider audience by being more discoverable. As of Feb 10th 2025 his post on LinkedIn had a respectable 750 impressions!
+### *Key Exploitable Results*
+We suggested Will add at least one Key Exploitable Result (KER) from his project to the KERs page; this is the place where you ensure that your results are known in the wider team and used beyond the scope of the project, for example by team leadership when scoping the creation of new work.
+The KER added was number 030; “A code base for reproducibly evaluating summaries using LLM-as-a-Judge”
+### *End result – Re-use!*
+The end result of Will’s efforts are that his project is know about in the wider team and beyond. This has even led to it being suggested for re-use in the Synthetic Data Generation project!
+
+## Will’s feedback on the engagement:
+*When we first asked you about filling in the marketing section of the quality framework, how apprehensive were you?* - "I didn't really think I was doing any marketing. I wasn't sure what to put in it; for example, I didn't recognise uploading code to GitHub as a form of marketing/comms".
+How helpful did you find talking to the comms and marketing team? - "incredibly helpful, went from not really knowing to understanding and recognising".
+*How long did it take you to fill in the quality framework pages and keep it up to date?* - "It was quite quick. It helped having spoken to Adam and Amaia. It wasn't arduous".
+
+## Do you need help with Comms and Marketing, or advice on presenting?
+Contact a member of the comms and marketing team (for example [adam.hollings1@nhs.net](adam.hollings1@nhs.net) or [Amaia.imazblanco1@nhs.net](Amaia.imazblanco1@nhs.net) ) and we can have a chat on how we can help you! 
```

Comment:
```suggestion
Contact a member of the comms and marketing team (for example [mailto:adam.hollings1@nhs.net](mailto:adam.hollings1@nhs.net) or [Amaia.imazblanco1@nhs.net](mailto:Amaia.imazblanco1@nhs.net) ) and we can have a chat on how we can help you! 
```
otherwise it thinks the link is a relative link on the website and breaks it.